cranelift/codegen/src/machinst/lower.rs

```diff
@@ -899,18 +908,21 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
     }
 
     /// Lower the function.
-    pub fn lower<B: LowerBackend<MInst = I>>(mut self, backend: &B) -> CodegenResult<VCode<I>> {
+    pub fn lower<B: LowerBackend<MInst = I>>(
+        mut self,
+        backend: &B,
+    ) -> CodegenResult<(VCode<I>, SigSet)> {
```

Comment:
Is there any way that we can stuff the `SigSet` into the `VCode`? It seems like maybe it is held separately here because of borrow-checker issues but maybe we can work around that if so.

Reply:
Done! Required a little bit of plumbing accessors through so things that have a `Lower` could access the `Lower`'s `VCodeBuilder`'s `VCode`'s `SigSet` but it was ultimately a nice clean up.